bin/generate-fixtures.js

```diff
@@ -16,14 +15,30 @@ if (!GITHUB_TOKEN) {
   )
 }
 
-const branches = [
-  'merge-commit',
-  'rebase-merging',
-  'squash-merging',
-  'overlapping-label',
+const repos = [
+  {
+    owner: 'TimonVS',
+    branch: 'merge-commit',
+  },
+  {
+    owner: 'TimonVS',
+    branch: 'rebase-merging',
+  },
+  {
+    owner: 'TimonVS',
+    branch: 'squash-merging',
+  },
+  {
+    owner: 'TimonVS',
+    branch: 'overlapping-label',
+  },
+  {
+    owner: 'jetersen',
+    branch: 'forking',
```

Comment:
Reason for this was because associated PRs does not immediately appear on the main repo or at least that was the behavior I saw.